app/src/features/surveys/view/SurveyObservations.tsx

```diff
@@ -286,39 +286,42 @@ const SurveyObservations: React.FC<ISurveyObservationsProps> = (props) => {
 
                   {submissionMessageTypes.length > 0 && (
                     <Box mt={1}>
-                      {submissionMessageTypes
-                        .sort((messageA, messageB) => {
-                          // Message A is sorted before B
-                          if (messageA.messageTypeLabel < messageB.messageTypeLabel) {
-                            return -1;
-                          }
-
-                          // Message B is sorted before A
-                          if (messageA.messageTypeLabel > messageB.messageTypeLabel) {
-                            return 1;
-                          }
-
-                          // Items are already in order
-                          return 0;
-                        })
-                        .map((messageType) => {
-                          return (
-                            <Box key={messageType.messageTypeLabel}>
-                              <Alert severity={alertSeverityFromSeverityLabel(messageType.severityLabel)}>
-                                {messageType.messageTypeLabel}
-                              </Alert>
-                              <Box component="ul" my={3}>
-                                {messageType.messages.map((messageObject: { id: number; message: string }) => {
-                                  return (
-                                    <li key={messageObject.id}>
-                                      <Typography variant="body2">{messageObject.message}</Typography>
-                                    </li>
-                                  );
-                                })}
+                      {
+                        // Alphabetize message types for consistency
+                        submissionMessageTypes
+                          .sort((messageA, messageB) => {
```

Comment:
I wonder if it would be better to do the sorting first in JS, outside of the return, and then this code can just iterate over the array as provided.

Reply:
mainly just thinking about how to keep the html simple

Reply:
good call, moved it into it's own function